app/src/org/commcare/dalvik/activities/HomeButtons.java

```diff
@@ -166,6 +172,7 @@ public void update(HomeCardDisplayData cardDisplayData,
     private static View.OnClickListener getLogoutButtonListener(final CommCareHomeActivity activity) {
         return new View.OnClickListener() {
             public void onClick(View v) {
+                reportButtonClick(GoogleAnalyticsFields.LABEL_LOGOUT_BUTTON);
```

Comment:
Is there any chance that we want to make a, like, 

"AnalyticsReportingOnClickListener()"

that we can use to handle this kind of thing a bit more generically? In theory we could use a few different strategies to get a String to Label the button